google/ggrc-core
# GGRC-7678/GGRC-7713 Change manual linking option for Issues and Assessments (remove Ticket tracker fields + change fields sync logic)

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] GGRC-7713

# Issue description

when creating an Issue object and linking it to existing ticket in Issue tracker, there are Component ID, Hotlist ID, Ticket Type, Priority, Severity and Title fields displayed. It's illogical to update these fields that have already  been set up in the existing Issue Tracker ticket.

# Steps to test the changes

Open the Audit
Click Map button in Issue tab
Click 'Map and create a new issue"
Switch 'ON' Issue Tracker and click 'Link to existing ticket' button
Check the displayed fields
Actual: Component ID, Hotlist ID, Ticket Type, Priority, Severity and Title fields are displayed

Expected: Component ID, Hotlist ID, Ticket Type, Priority, Severity and Title fields should be removed from "Link to existing ticket". Only a mandatory 'Ticket ID' field should be displayed. Ticket Priority, Ticket Severity, Ticket title should be synced from Ticket tracker.

# Solution description

Fetch IssueTracker data from api and update object issue using it. 

# Sanity checklist




# PR Review checklist
